jobs/ostree-compose.yml

```diff
@@ -59,40 +59,30 @@
               which repo of the package being updated
     builders:
         - shell: |
-            cat << EOF > ${WORKSPACE}/job.properties_running
-                topic=org.centos.prod.ci.pipeline.compose.running
-                build_url=$BUILD_URL
-                compose_url=http://artifacts.ci.centos.org/artifacts/fedora-atomic/${fed_branch}/ostree
-                compose_rev=
-                build_id=$BUILD_ID
-                ref=fedora/${fed_branch}/x86_64/atomic-host
-                rev=${rev}
-                namespace=rpms
-                repo=$fed_repo
-                status=success
-                test_guidance=
-            EOF
+            if [ "$fed_branch" = "master" ]; then
+                echo "fed_branch=rawhide" > ${WORKSPACE}/job.properties
+            fi
         - inject:
-            properties-file: ${WORKSPACE}/job.properties_running
+            properties-file: ${WORKSPACE}/job.properties
         - jms-messaging:
             override-topic: org.centos.prod.ci.pipeline.compose
```

Comment:
why do you echo everything out to a properties file only to read it back in?  Isn't this exactly what you complained about in this PR of mine? Don't get me wrong, I agree with your assessment and that's why I changed it.